scripts/ci/lib.sh

```diff
@@ -1252,9 +1252,8 @@ send_slack_notice_for_failures_on_merge() {
         return 0
     fi
 
-    local tag
-    tag="$(make --quiet --no-print-directory tag)"
-    if [[ "$tag" =~ $RELEASE_RC_TAG_BASH_REGEX ]]; then
```

Comment:
Note: it was already skipped for a release build process that combined cherry-picks and tagging in one push.